### PR TITLE
Prep for architect sandbox service discovery

### PR DESCRIPTION
**Pull request by kristoferjoseph**

### CODE REVIEW

File: src/helpers/_get-table-name-sandbox.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = async function getTableName () {
   if (process.env.ARC_ENV === 'testing' ||
       process.env.NODE_ENV === 'testing') {
     try {
-      let result = await tiny.get({ url: '/_asdf' })
+      let result = await tiny.get({ url: '/_asd' })
```
Comment (ryanblock) [inline src/helpers/_get-table-name-sandbox.js:12]: Ha! I def thought it was funny, why I suggested it! But it's probably also obscure. ¯\_(ツ)_/¯

File: src/helpers/_get-table-name-sandbox.js [hunk 2]
```diff
@@ -1,46 +1,48 @@
...
+      // attempt to read the arcfile
+      let { arc } = read({ type, cwd })
 
-  let { arc } = result
-  if (arc.tables && arc.tables.some(t => Object.keys(t)[0] === 'data'))
-    return `${arc.app[0]}-staging-data`
-  throw ReferenceError('@tables not defined! begin/data expects a data table')
-  */
+      // look for a table named 'data'
+      if (arc.tables && arc.tables.some(t => Object.keys(t)[0] === 'data')) {
```
Comment (ryanblock) [inline src/helpers/_get-table-name-sandbox.js:35]: If we have Inventory in Begin Data, why using the manifest reader? Imo best practice here would be to run Inventory.

Comment (kristoferjoseph) [reply src/helpers/_get-table-name-sandbox.js:35]: Good question!
I think there was a reason @brianleroux needed to just use `read`

File: test/integration/namespace-test.js [hunk 3]
```diff
@@ -35,5 +35,5 @@ test('shutdown sandbox', async t=> {
 process.on('unhandledRejection', (reason, p) => {
   console.log(reason)
   console.log(p)
-  end()
+  sandbox.end()
```
Comment (kristoferjoseph) [inline test/integration/namespace-test.js:38]: This doesn't work. Sandbox keeps running, but the code that was previously here was using the outdated `end()` syntax that threw an unrelated error when an unhandled promise rejection happened.

**Pull request merged**